projects/js-packages/charts/tsup.config.ts

```diff
@@ -1,23 +1,33 @@
+import { sassPlugin, postcssModules } from 'esbuild-sass-plugin';
 import { defineConfig } from 'tsup';
 import pkg from './package.json';
 
 // Extract entries from package exports
-const entry = Object.values( pkg.exports ).map( ( { 'jetpack:src': src } ) => src );
+const entry = Object.values( pkg.exports )
+	.map( $export => ( typeof $export === 'object' ? $export[ 'jetpack:src' ] : null ) )
```

Comment:
Let us use an empty string to keep types consistent.

```suggestion
	.map( $export => ( typeof $export === 'object' ? $export[ 'jetpack:src' ] : '' ) )
```

Reply:
Well, never mind. You already added a type guard

Reply:
I added `path is string` but should be alright. Thanks!